app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SessionsController < ApplicationController
+      # TODO: samuel - Bypass CSRF token for now
+      skip_before_action :verify_authenticity_token
+
+      # POST /api/v1/sessions
+      def create
+        user = User.find_by(email: session_params[:email])
+
+        if user.present? && user.authenticate(session_params[:password])
+          sign_in user
+          render json: { current_user:, signed_in: true }, status: :ok
+        else
+          render json: { signed_in: false }, status: :bad_request
+        end
+      end
+
+      # DELETE /api/v1/sessions/:id
+      def destroy
+        session[:user_id] = nil
+        render json: { signed_in: false }, status: :ok
+      end
+
+      # GET /api/v1/sessions/signed_in
+      def signed_in
+        if current_user
+          render json: { current_user:, signed_in: true }
+        else
+          render json: { signed_in: false }
+        end
+      end
```

Comment:
Why not do this as `index`?

Reply:
Indeed, we could make use of the default route.